.idea/opg-refunds.iml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/public-front/vendor/alphagov/notifications-php-client" />
```

Comment:
Do we know why all these excludes exist?

Reply:
not a clue - i can remove this if needed.

Reply:
Yeah. I think if you look at the PR that went into Use these were removed at somepoint.